scripts/secnetperf.ps1

```diff
@@ -111,12 +111,15 @@ INSERT OR IGNORE INTO Secnetperf_builds (Secnetperf_Commit, Build_date_time, TLS
 VALUES ('$MsQuicCommit', '$(Get-Date -Format "yyyy-MM-dd HH:mm:ss")', 1, 'TODO');
 "@
 $json = @{}
-$allTests = @(
-    "-exec:maxtput -up:10s -ptput:1",
-    "-exec:maxtput -down:10s -ptput:1",
-    "-exec:maxtput -rconn:1 -share:1 -conns:100 -run:10s -prate:1",
-    "-exec:lowlat -rstream:1 -up:512 -down:4000 -run:10s -plat:1"
-)
+
+$allTests = [System.Collections.Specialized.OrderedDictionary]::new()
+
+# > All tests:
+$allTests["maxtput-up-1"] = "-exec:maxtput -up:10s -ptput:1"
```

Comment:
What is the `1` signify here?

Reply:
So Instead of integer IDs, we have string IDs, and the 1 just differentiates between other maxtput tests we might add in the future with different arguments (up: 20s) or something.

Reply:
I'm not so sure about this. Let's talk some.